Makefile

```diff
@@ -0,0 +1,6 @@
+all: README.md
+
+README.md: README.md.njk render-template.ts Makefile data/projects.toml
```

Comment:
What is the purpose of having a make file invoke the render script? This seems like an extra level of complexity that could be directly in the action yaml.

Reply:
The makefile is to ensure that when rebuilding README.md I run the same command locally on my computer and in GitHub Actions. It is also to ensure the command stays the same between invocations. Since two copies of a command in two places can diverge, the command should be either in the task runner/build tool or in the action. I could choose the latter instead and use act to run the action itself locally in Docker, but it would be slower, and I don't think the isolation provided by Docker is necessary here. Make(1) will probably be easier for contributors using Linux (including WSL) and macOS to preview their changes, though it is not a very important factor due to the action.